lib/evaluate.js

```diff
@@ -133,32 +135,60 @@ async function executeAsync(yargs, argv) {
 
     let evaluateResults
     try {
-      evaluateResults = await EvaluateProofs(proofs)
+      evaluateResults = await EvaluateProofs(proofs, btc)
     } catch (error) {
       console.error(`ERROR : Could not evaluate proofs`)
       return
     }
 
-    displayOutputResults(evaluateResults, output, quiet, json)
+    displayOutputResults(evaluateResults, output, quiet, json, btc)
   }
 }
 
-function displayOutputResults(resultsArray, outputObj, quiet, json) {
-  for (let x = 0; x < resultsArray.length; x++) {
-    outputObj.addSuccessResult({
-      hash_id_node: resultsArray[x].hash_id_node,
-      type: resultsArray[x].type,
-      anchor_id: resultsArray[x].anchor_id,
-      expected_value: resultsArray[x].expected_value
-    })
+function displayOutputResults(resultsArray, outputObj, quiet, json, btc) {
+  for (let result of resultsArray) {
+    let tx, output
+    output = {
+      hash_id_node: result.hash_id_node,
+      type: result.type,
+      anchor_id: result.anchor_id,
+      expected_value: result.expected_value
+    }
+    if (btc && result.raw_btc_tx) {
+      tx = TX.fromRaw(result.raw_btc_tx, 'hex')
+      // 1 prevout should be sufficient to verify anchor
+      // but show the count to provide additional information
+      output.prevout_count = tx.inputs.length
+      output.prevout_1 = tx.inputs[0].prevout.rhash()
+    }
+    outputObj.addSuccessResult(output)
   }
   outputObj.display(quiet, json)
 }
 
-async function EvaluateProofs(proofs) {
-  let evaluateResponse
+async function EvaluateProofs(proofs, btc) {
```

Comment:
**Note**: This function is a little more complex than before since `evaluateProofs` and `getProofTxs` come back differently and we need to merge the btc information only when there is a match in the `evaluateProofs` response. 

Also note that there is a nested loop in the form of the `findIndex` but from what I've seen, this will only ever be looping through an array of 1 or 2 items. I imagine this might increase as other anchors are supported but not to a point where the nest loop severely affects performance.